### PR TITLE
Update copy in dapp suggested gas fee tooltip to: "Edit to use MetaMask's recommended gas fee based on the latest block." #11952

### LINKED ISSUE #11952: Update copy in dapp suggested gas fee tooltip to: "Edit to use MetaMask's recommended gas fee based on the latest block."
For EIP-1559 dapp suggested gas fees, we need three updates:

1. Change the `[domain] suggested gas fee (i)` to `Estimated gas fee (i)`
2. Change that line's color from orange to black
3. Under that line, add `Site suggested` in italics in gray.

image (1)

### PR DESCRIPTION
**Pull request by segun**
Fixes: #11952 
    Change the [domain] suggested gas fee (i) to Estimated gas fee (i)
    Change that line's color from orange to black
    Under that line, add Site suggested in italics in gray.
Explanation:  

Manual testing steps:  
  - 
  - 
  -

### CODE REVIEW

**Comment by segun** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by danjm** [comment]
looks like you can run `yarn lint:fix` locally to fix the lint issues

**Review by danjm — DISMISSED** [review]
Code looks good and I tested in the browser and it looks correct. Just need to correct the lint issues.

**Review by Jochy1873 — DISMISSED** [review]

File: app/_locales/en/messages.json [hunk 1]
```diff
@@ -2515,9 +2515,8 @@
   "transactionCreated": {
     "message": "Transaction created with a value of $1 at $2."
   },
-  "transactionDetailDappGasHeading": {
-    "message": "$1 suggested gas fee",
-    "description": "$1 represents a dapp origin"
+  "transactionDetailDappGasMoreInfo": {
+    "message": "Site suggested"
```
Comment (adonesky1) [inline app/_locales/en/messages.json:2519]: Wondering if we want this to be "dapp suggested" or something along those lines?

**Review by adonesky1 — DISMISSED** [review]
Great work Olu! Really cool you've added the font-style options to the design system! I had just one non-blocking question about the wording of the copy change. But everything looks and works great! Just need to fix the lint issues and this should be good to go!

File: app/_locales/en/messages.json (hunk 1 above)
Comment (adonesky1) [reply app/_locales/en/messages.json:2519]: @rachelcope ^

**Review by ryanml — APPROVED** [review]
looks great! Thanks for fixing the lint issues!

**Review by adonesky1 — APPROVED** [review]

**Pull request merged**